.github/workflows/ci.yml

```
@@ -63,18 +64,26 @@ jobs:
     
   testing:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       
-      - name: Set up Python
-        uses: actions/setup-python@v5
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)